pkg/token/token_test.go

```diff
@@ -13,16 +13,25 @@ import (
 )
 
 func validationErrorTest(t *testing.T, token string, expectedErr string) {
+	t.Helper()
 	_, err := tokenVerifier{}.Verify(token)
 	errorContains(t, err, expectedErr)
 }
 
 func errorContains(t *testing.T, err error, expectedErr string) {
+	t.Helper()
 	if err == nil || !strings.Contains(err.Error(), expectedErr) {
 		t.Errorf("err should have contained '%s' was '%s'", expectedErr, err)
 	}
 }
 
+func isSTSError(t *testing.T, err error) {
```

Comment:
Nit: I'd prefer validateSTSError or assertSTSError because you aren't returning a bool, your asserting and erroring on failure.

Reply:
Done.